test/specs/core.controller.tests.js

```diff
@@ -125,26 +125,6 @@ describe('Chart', function() {
 			defaults.line.spanGaps = false;
 		});
 
-		it('should override axis positions that are incorrect', function() {
```

Comment:
can we add this back at all? I realize this might have been removed from my original MR but I think we can support it still in this form

Reply:
Good catch, added the test back!